lib/Notification/Listener.php

```diff
@@ -275,11 +275,11 @@ protected function sendCallNotifications(Room $room): void {
 		}
 
 		$this->preparedCallNotifications = [];
-		$userIds = $this->participantsService->getParticipantUserIdsForCallNotifications($room);
+		$users = $this->participantsService->getParticipantUsersForCallNotifications($room);
```

Comment:
```suggestion
        $userIds = $this->participantsService->getParticipantUsersForCallNotifications($room);
```

or rename the variable in L291

Reply:
Have to take array keys in 291.
Fixed